src/transactions/slice.ts

```diff
@@ -246,7 +246,7 @@ const slice = createSlice({
              * - ignore pending as it should only affect confirmed transactions that are already
```

Comment:
should this comment also be removed? i don't really understand what it means, but it seems like the pending transactions were intentionally checked before

Reply:
Ah, you're right! Forgot to remove, sorry!

Reply:
@kathaypacific fixed!